code/modules/client/client_procs.dm

```diff
@@ -422,10 +422,12 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	if( (world.address == address || !address) && !GLOB.host )
 		GLOB.host = key
 		world.update_status()
-
+	
 	if(holder)
 		add_admin_verbs()
-		to_chat(src, get_message_output("memo"))
+		var/admin_memo_note = get_message_output("memo")
+		if(admin_memo_note)
+			to_chat(src, get_message_output("memo"))
```

Comment:
why are we running get_message_output a second time here

Reply:
oh true